app/serializers/page_serializer/home_page_serializer.rb

```diff
@@ -0,0 +1,79 @@
+module PageSerializer
+  class HomePageSerializer < PageSerializer::BasePageSerializer
```

Comment:
Don't hard code any links - use a route function with a SPARQL query to get the IDs for collections / houses / parties etc

Reply:
Can this be done in the next pull request? It might be neater and more manageable having two smaller requests rather than on massive one.